src/shared/components/TimeWidget.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC, useState } from "react";
+import { WidgetProps } from "@rjsf/core";
+import { format, isValid } from "date-fns";
+import MuiTimePicker from "@material-ui/lab/TimePicker";
+import AdapterDateFns from "@material-ui/lab/AdapterDateFns";
+import { LocalizationProvider } from "@material-ui/lab";
+import { TextFieldProps } from "@material-ui/core";
+import { TextWidget } from "./TextWidget";
+
+// TODO: Fix typings for TextWidget
+export const TimeWidget: FC<WidgetProps> = (props: any) => {
+  const { onChange, value } = props;
+  const [time, setTime] = useState(value);
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
+      <MuiTimePicker
```

Comment:
It troubles me that the font is incorrect and the new TimePicker doesn't support classes or styles.
I'll have a look at overriding the global style.. maybe..